src/musesounds/qml/MuseScore/MuseSounds/MuseSoundsPage.qml

```diff
@@ -0,0 +1,189 @@
+    }
+
+    Column {
+        id: topLayout
+
+        anchors.top: parent.top
+        anchors.topMargin: prv.sideMargin
+        anchors.left: parent.left
+        anchors.leftMargin: prv.sideMargin
+        anchors.right: parent.right
+        anchors.rightMargin: prv.sideMargin
+
+        spacing: 24
+
+        StyledTextLabel {
+            id: pageTitle
+            Layout.fillWidth: true
+
+            text: qsTrc("appshell", "Muse Sounds")
+            font: ui.theme.titleBoldFont
+            horizontalAlignment: Text.AlignLeft
+        }
+    }
+
+    GradientRectangle {
+        id: topGradient
+
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.top: flickable.top
+
+        startColor: root.color
+        endColor: "transparent"
+    }
+
+    StyledFlickable {
+        id: flickable
+
+        anchors.top: topLayout.bottom
+        anchors.topMargin: 24
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.bottom: parent.bottom
+
+        contentWidth: width
+        contentHeight: column.implicitHeight
+
+        topMargin: topGradient.height
+        bottomMargin: 24
+
+        Column {
+            id: column
+
+            anchors.fill: parent
+            anchors.leftMargin: prv.sideMargin
+            anchors.rightMargin: prv.sideMargin
+
+            spacing: 22
+
+            Repeater {
+                model: museSoundsModel
+
+                delegate: MuseSoundsListView {
+                    width: parent.width
+
+                    title: modelData.title
+                    visible: count > 0
+
+                    model: modelData.sounds
+
+                    flickableItem: column
+
+                    navigationPanel.section: navSec
+                    navigationPanel.name: title + "Sounds"
+                    navigationPanel.order: index
+
+                    onGetSoundRequested: {
+                        // todo
+                    }
+
+                    onNavigationActivated: function(itemRect) {
+                        Utils.ensureContentVisible(flickable, itemRect, headerHeight + 16)
+                    }
+                }
+            }
+        }
+    }
+
+    Column {
+        id: errorMessageColumn
+        visible: museSoundsModel.isEmpty
+
+        anchors.top: parent.top
+        //! NOTE: should be synchronized with the error message from Learn page
+        anchors.topMargin: 127 + topGradient.height + Math.max(parent.height / 3 - height / 2, 0)
+        anchors.left: parent.left
+        anchors.right: parent.right
+
+        spacing: 16
+
+        StyledTextLabel {
+            width: parent.width
+            font: ui.theme.tabBoldFont
+            text: qsTrc("musesounds", "Sorry, we are unable to load these sounds right now")
+        }
+
+        StyledTextLabel {
+            width: parent.width
+            text: qsTrc("musesounds", "Please check your internet connection or try again later.")
```

Comment:
We have the same string in many other places... Perhaps the context should be changed to "global" (at least here)